package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "qpp-measures-data",
-  "version": "5.0.0",
+  "version": "3.1.0-beta.1",
```

Comment:
Small typo in the version number but that can be corrected when the non-beta package is published.